contact.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link href="css/style.css" rel="stylesheet" type="text/css">
+    <link href="https://fonts.googleapis.com/css?family=Anton|Cinzel|Exo:700|Hind+Siliguri|K2D|Pathway+Gothic+One&display=swap" rel="stylesheet">
+    <title>Contact</title>
+</head>
+<body>
+  <div class="page-container">
+    <div class="content-wrap">
+      <ul>
+        <li><a class ="active" href="index.html">Home</a></li>
+        <li><a href="contact.html">Contact</a></li>
+        <li><a href="portfolio.html">Portfolio</a></li>
+      </ul>
+      <div class="title-info">
+        <h1>Contact Me</h1>
+      </div>
+      <div class=form>
+        <form action="/action_page.php"> <!--what does the /action_page.php do-->
+        Name:<br>
```

Comment:
Indentation is off

Reply:
I corrected the indentation on line 24 to like 35.